CHANGELOG.md

```diff
@@ -1,7 +1,23 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
-The format is based on [Keep a Changelog](http://keepachangelog.com/) 
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+
+## [8.2.0] Stable Release
+### Added
+- Added new tests for Always Encrypted for secure enclaves feature [#1166](https://github.com/microsoft/mssql-jdbc/pull/1166)
+- Added backwards compatibility for calling SQLServerClob.length() on nvarchar columns [#1214](https://github.com/microsoft/mssql-jdbc/pull/1214)
+
+### Fixed issues
+- Fixed an issue with potentially creating more enclave sessions than needed [#1208](https://github.com/microsoft/mssql-jdbc/pull/1208)
+- Fixed an issue with InputStream closing when calling SQLServerBlob.leangth() on an `image` column [#1214](https://github.com/microsoft/mssql-jdbc/pull/1214)
```

Comment:
.length()

Reply:
There were 2 issues with Blob.

1. Fixed an issue when calling Blob.length() on an image column while streaming would close the stream.
2. Fixed an issue when calling Clob.length() on a nvarchar column would not return the correct character count.

They're both in #1214